awx/main/utils/formatters.py

```diff
@@ -97,7 +97,7 @@ def format_exception(cls, exc_info):
 
     @classmethod
     def serialize(cls, message):
-        return bytes(json.dumps(message, cls=DjangoJSONEncoder), 'utf-8')
+        return json.dumps(message, cls=DjangoJSONEncoder) + '\n'
```

Comment:
Curious, why the `\n` ?

Reply:
https://stackoverflow.com/a/52957869

Reply:
(working around a bug we noticed in the cpython syslog handler code)